benchflow-dsl/src/main/scala/cloud/benchflow/dsl/definition/configuration/BenchFlowExperimentConfigurationYamlProtocol.scala

```diff
@@ -16,6 +16,10 @@ import scala.util.Try
  */
 object BenchFlowExperimentConfigurationYamlProtocol extends DefaultYamlProtocol {
 
+  // default value since deserializing from a test definition it might not be defined
+  // before use it will be overridden
+  val defaultUsers = 0
```

Comment:
When creating an experiment from a test definition (the only way we support) we need to provide a default value for the users since it is optional in the test but not optional in the experiment (we always want it to be defined since otherwise it cannot be executed). The default value is more of a temporary placeholder since it has to be overridden when building the experiment using the configuration of the point in the exploration space. If the exploration space does not define  users then the semantic check should have thrown an error earlier.